app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
+end
```

Comment:
- [ ] Again here just as we have to do in the post model, kindly add a rails callback  using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to database, then make the `update_comments_counter` method private [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new post has been saved to database

e.g

```ruby
after_save :update_comments_counter

  private

  def update_comments_counter
    author.increment!(:comments_counter)
  end
```